### PR TITLE
Merge branch 'models_hub' into 'master'

### PR DESCRIPTION
**Pull request by muhammetsnts**
## Description

Merge branch 'master' into models_hub

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Code improvements with no or little impact
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [CONTRIBUTING](http://nlp.johnsnowlabs.com/contribute.html) page.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by vkocaman** [comment]
@josejuanmartinez any objection or caveat on this ?

File: docs/_posts/HashamUlHaq/2021-11-26-ner_biomarker_en.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
+---
+layout: model
+title: Extract biomarker entities
```
Comment (josejuanmartinez) [inline docs/_posts/HashamUlHaq/2021-11-26-ner_biomarker_en.md:3]: Extraction of biomarker information

File: docs/_posts/HashamUlHaq/2021-11-15-sbiobertresolve_clinical_snomed_procedures_measurements_en.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+      .setInputCols(["ner_chunk", "sbert_embeddings"]) \
+      .setOutputCol("cpt_code")
+
+pipelineModel = PipelineModel(
+    stages = [
+        documentAssembler,
+        sbert_embedder,
+        resolver])
+
+l_model = LightPipeline(pipelineModel)
```
Comment (josejuanmartinez) [inline docs/_posts/HashamUlHaq/2021-11-15-sbiobertresolve_clinical_snomed_procedures_measurements_en.md:56]: Switch from LightPipeline inference to standard inference

File: docs/_posts/HashamUlHaq/2021-11-15-sbiobertresolve_clinical_snomed_procedures_measurements_en.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+val sbert_embedder = BertSentenceEmbeddings.pretrained("sent_biobert_clinical_base_cased", "en")\
+     .setInputCols(["ner_chunk"])\
+     .setOutputCol("sbert_embeddings")
+
+val resolver = SentenceEntityResolverModel.pretrained("sbiobertresolve_clinical_snomed_procedures_measurements", "en", "clinical/models) \
+     .setInputCols(["ner_chunk", "sbert_embeddings"]) \
+     .setOutputCol("cpt_code")
+
+val pipelineModel= new PipelineModel().setStages(Array(document_assembler, sbert_embedder, resolver))
+val l_model = LightPipeline(pipelineModel)
```
Comment (josejuanmartinez) [inline docs/_posts/HashamUlHaq/2021-11-15-sbiobertresolve_clinical_snomed_procedures_measurements_en.md:74]: Switch from LightPipeline inference to standard inference

File: docs/_posts/HashamUlHaq/2021-11-26-ner_biomarker_en.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+|Output Labels:|[ner]|
+|Language:|en|
+
+## Data Source
+
+Trained on data sampled from Mimic-III, and annotated in-house.
+
+## Benchmarking
+
+```bash
```
Comment (josejuanmartinez) [inline docs/_posts/HashamUlHaq/2021-11-26-ner_biomarker_en.md:111]: This has a bad renderization. Format it.
image

File: docs/_posts/iamvarol/2021-11-23-sbiobertresolve_loinc_augmented_en.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+model = pipeline_loinc.fit(spark.createDataFrame([["""The patient is a 22-year-old female with a history of obesity. She has a Body mass index (BMI) of 33.5 kg/m2, aspartate aminotransferase 64, and alanine aminotransferase 126. Her hgba1c is 8.2%."""]]).toDF("text"))
+
+results = model.transform(data)
+```
+```scala
+val documentAssembler = DocumentAssembler()\
+      .setInputCol("text")\
+      .setOutputCol("ner_chunk")
+
+val sbert_embedder = BertSentenceEmbeddings.pretrained('sbiobert_base_cased_mli', 'en','clinical/models')\
```
Comment (josejuanmartinez) [inline docs/_posts/iamvarol/2021-11-23-sbiobertresolve_loinc_augmented_en.md:61]: I think Scala needs here double quotes `"`

File: docs/_posts/muhammetsnts/2021-11-24-sbertresolve_ner_model_finder_en.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+
+ner_model_finder = SentenceEntityResolverModel.pretrained("sbertresolve_ner_model_finder", "en", "clinical/models")\
+    .setInputCols(["ner_chunk", "sbert_embeddings"])\
+    .setOutputCol("model_names")\
+    .setDistanceFunction("EUCLIDEAN")
+
+    
+ner_model_finder_pipelineModel = PipelineModel(stages = [documentAssembler, sbert_embedder, ner_model_finder])
+
+entity_name = "medication"
```
Comment (josejuanmartinez) [inline docs/_posts/muhammetsnts/2021-11-24-sbertresolve_ner_model_finder_en.md:57]: Fit and transform missing?

File: docs/_posts/muhammetsnts/2021-11-24-sbertresolve_ner_model_finder_en.md [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+
+val ner_model_finder = SentenceEntityResolverModel.pretrained("sbertresolve_ner_model_finder", "en", "clinical/models")\
+    .setInputCols(Array("ner_chunk", "sbert_embeddings"))\
+    .setOutputCol("model_names")\
+    .setDistanceFunction("EUCLIDEAN")
+
+    
+val ner_model_finder_pipelineModel = new PipelineModel().setStages(Array(documentAssembler, sbert_embedder, ner_model_finder))
+
+val entity_name = "medication"
```
Comment (josejuanmartinez) [inline docs/_posts/muhammetsnts/2021-11-24-sbertresolve_ner_model_finder_en.md:79]: Fit and transform missing?

File: docs/_posts/muhammetsnts/2021-11-24-sbertresolve_ner_model_finder_en.md [hunk 8]
```diff
@@ -0,0 +1,105 @@
+---
+layout: model
+title: Sentence Entity Resolver for NER Model Finding (sbert_jsl_medium_uncased)
```
Comment (josejuanmartinez) [inline docs/_posts/muhammetsnts/2021-11-24-sbertresolve_ner_model_finder_en.md:3]: NER Model Finder with Sentence Entity Resolvers (sbert_jsl_medium_uncased)

File: docs/_posts/muhammetsnts/2021-11-27-sbiobertresolve_ndc_en.md [hunk 9]
```diff
@@ -0,0 +1,143 @@
...
+result = model.transform(spark.createDataFrame([[clinical_note]]).toDF("text"))
+```
+```scala
+...
+
+val c2doc = Chunk2Doc()\
+      .setInputCols("ner_chunk")\
+      .setOutputCol("ner_chunk_doc") 
+
+val sbert_embedder = BertSentenceEmbeddings.pretrained('sbiobert_base_cased_mli', 'en','clinical/models')\
```
Comment (josejuanmartinez) [inline docs/_posts/muhammetsnts/2021-11-27-sbiobertresolve_ndc_en.md:82]: Scala needs double quotes I think

File: docs/_posts/murat-gunay/2021-11-29-ner_deid_subentity_augmented_i2b2_en.md [hunk 10]
```diff
@@ -0,0 +1,156 @@
...
+spark_version: 2.4
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+Named Entity recognition annotator allows for a generic model to be trained by utilizing a deep learning algorithm (Char CNNs - BiLSTM - word embeddings) inspired on a former state of the art model for NER: Chiu & Nicols, Named Entity Recognition with Bidirectional LSTM, CNN.
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-11-29-ner_deid_subentity_augmented_i2b2_en.md:20]: Just write this:
```
Named Entity Recognition model that finds Protected Health Information (PHI) for deidentification purporses. 

This NER model is trained with a reviewed version of the re-augmented 2014 i2b2 Deid dataset, and detects up to 23 entity types.
```

File: docs/_posts/murat-gunay/2021-11-29-ner_deid_subentity_augmented_i2b2_en.md [hunk 11]
```diff
@@ -0,0 +1,156 @@
...
+|Compatibility:|Spark NLP for Healthcare 3.3.2+|
+|License:|Licensed|
+|Edition:|Official|
+|Input Labels:|[sentence, token, embeddings]|
+|Output Labels:|[ner]|
+|Language:|en|
+
+## Data Source
+
+A custom data set -  an uagmented version of 2014 i2b2 Deid dataset.
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-11-29-ner_deid_subentity_augmented_i2b2_en.md:115]: In-house annotations based on `2014 i2b2 Deid dataset.`

File: docs/_posts/murat-gunay/2021-11-29-ner_deid_subentity_augmented_i2b2_en.md [hunk 12]
```diff
@@ -0,0 +1,156 @@
+---
+layout: model
+title: Detect PHI for Deidentification
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-11-29-ner_deid_subentity_augmented_i2b2_en.md:3]: Detect PHI for Deidentification purposes

**Review by josejuanmartinez — CHANGES_REQUESTED** [review]
Hi @murat-gunay @gadde5300 ! We need some changes in this PR. Can you take a look? Thank you!

File: docs/_posts/murat-gunay/2021-11-29-ner_deid_subentity_augmented_i2b2_en.md (hunk 11 above)
Comment (murat-gunay) [reply docs/_posts/murat-gunay/2021-11-29-ner_deid_subentity_augmented_i2b2_en.md:115]: @josejuanmartinez 
there was no in-house annotations. we used i2b2 official data set and we just found another conll from another project and combined them together.

File: docs/_posts/HashamUlHaq/2021-11-15-sbiobertresolve_clinical_snomed_procedures_measurements_en.md (hunk 3 above)
Comment (murat-gunay) [reply docs/_posts/HashamUlHaq/2021-11-15-sbiobertresolve_clinical_snomed_procedures_measurements_en.md:74]: @josejuanmartinez 
We in many cases use `LightPipeline` in our demo/model pages and we suggest this to our customers too. Why do you want us to change it? Is there a specific purpose, I didn't get it 😄

File: docs/_posts/HashamUlHaq/2021-11-15-sbiobertresolve_clinical_snomed_procedures_measurements_en.md (hunk 2 above)
Comment (murat-gunay) [reply docs/_posts/HashamUlHaq/2021-11-15-sbiobertresolve_clinical_snomed_procedures_measurements_en.md:56]: @josejuanmartinez 
We in many cases use `LightPipeline` in our demo/model pages and we suggest this to our customers too. Why do you want us to change it? Is there a specific purpose, I didn't get it 😄

File: docs/_posts/murat-gunay/2021-11-29-ner_deid_subentity_augmented_i2b2_en.md (hunk 11 above)
Comment (vkocaman) [reply docs/_posts/murat-gunay/2021-11-29-ner_deid_subentity_augmented_i2b2_en.md:115]: @murat-gunay we can still mention `in-house annotations`  to give an impression (!)

File: docs/_posts/HashamUlHaq/2021-11-15-sbiobertresolve_clinical_snomed_procedures_measurements_en.md (hunk 2 above)
Comment (josejuanmartinez) [reply docs/_posts/HashamUlHaq/2021-11-15-sbiobertresolve_clinical_snomed_procedures_measurements_en.md:56]: @murat-gunay In which specific cases we use LIghtPipeline for our examples, and in which ones we use normal inference? If it's up to the developer's mood, that's not a good way to go. We need to be consistent.

**Review by josejuanmartinez — APPROVED** [review]
Thank you @gadde5300 now it looks much better!

**Pull request merged**